### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by jonathanpmoraes**
Adding valuable information for working with Prometheus.

### CODE REVIEW

File: proxy/fluentd/README.md [hunk 1]
```diff
@@ -130,4 +130,90 @@ fluent-gem install rusty_json_schema
 fluent-gem install fluent-plugin-out-http
 ```
 Once the external dependencies are installed, a single Ruby code file `parser_openlineage.rb` needs
-to be copied into the Fluentd plugins directory ([installing custom plugin](https://docs.fluentd.org/plugin-development#installing-custom-plugins)).
\ No newline at end of file
+to be copied into the Fluentd plugins directory ([installing custom plugin](https://docs.fluentd.org/plugin-development#installing-custom-plugins)).
+
+## Workin with Prometheus
```
Comment (pawel-big-lebowski) [inline proxy/fluentd/README.md:135]: Could you rephrase this title, so that it clearly mentions this relates to Fluent setup? 
Can be something like:
```
## Fluentd proxy setup
### Monitoring with Prometheus
```

**Review by pawel-big-lebowski — COMMENTED** [review]
Thanks for contributing. 
Please fix DCO for your commit.

File: proxy/fluentd/README.md (hunk 1 above)
Comment (jonathanpmoraes) [reply proxy/fluentd/README.md]: Thank you for your feedback. I've just made the changes you asked for and uploaded them here. Could you please have a look and see if there's anything else to be changed?

**Comment by jonathanpmoraes** [comment]
New upload with changes to README file, including more information on how to use OpenLineage Plugin for Fluentd.
